rero/rero-ils
# invenio: upgrade to invenio circulation v1.0.0a26

This version fixes a problem of requests with concurrency.
RERO circulation can now making several renewals whitout any errors.

Co-Authored-by: Olivier DOSSMANN <git@dossmann.net>

## Why are you opening this PR?

To fix problems of renewals in https://github.com/rero/rero-ils-ui/pull/308

## Dependencies

* https://github.com/rero/rero-ils-ui/pull/308

## How to test?

After a setup of this branch, check on https://github.com/rero/rero-ils-ui/pull/308 (https://localhost:4200/) that you can make several renewals ('Renew all' button).

## Code review check list
